__tests__/TreeNode.test.js

```diff
@@ -7,89 +7,103 @@ import defaultAnimations from '../src/themes/animations';
-            simulateClickOnButton(wrapper);
-            expect(clickFn.mock.calls[0][1]).toBe(false);
+    describe('when NodeHeader is clicked', () => {
+        it('should call onToggle with the selected node and toggled in false', () => {
+            const wrapper = shallow(renderComponent());
+            simulateClickOnHeader(wrapper);
+            expect(onToggle).toHaveBeenCalled();
+            expect(onToggle).toBeCalledWith(data, false);
         });
     });
-    describe('Drawer', () => {
-        test('should children size to be equal to zero when toggled is false', () => {
-            const wrapper = shallow(renderComponent({node: {...data, toggled: false}}));
-            expect(wrapper.find('Drawer').children().length).toBe(0);
+
+    describe('> <Drawer/>', () => {
+        describe('when toggle is false', () => {
+            it('should have children.size to be 0', () => {
+                const wrapper = shallow(renderComponent({
+                    node: {...data, toggled: false}
+                }));
+                expect(wrapper.find('Drawer').children().length).toBe(0);
+            });
         });
-        test('should return Loading component when node has property loading in true', () => {
-            const wrapper = shallow(renderComponent({node: {id: 1, name: 'test', toggled: true, loading: true}}));
-            const drawer = wrapper.find('Drawer');
-            expect(drawer.find('Loading').exists()).toBe(true)
+
+        describe('when node has property loading in true', () => {
+            it('should render a Loading component', () => {
+                const wrapper = shallow(renderComponent({
+                    node: {id: 1, name: 'test', toggled: true, loading: true}
+                }));
+                const drawer = wrapper.find('Drawer');
+                expect(drawer.find('Loading').exists()).toBe(true);
+            });
         });
-        test('should return seven TreeNode children', () => {
+
+        it('should return seven TreeNode children', () => {
             const wrapper = shallow(renderComponent());
             const drawer = wrapper.find('Drawer');
             const ul = drawer.children();
             expect(ul.children()).toHaveLength(7);
-        })
+        });
     });
-    describe('animations method', () => {
-        describe('animations and node toggled are false', () => {
-            test('should return a toggle object with an animation with rotateZ 0 and duration zero', () => {
-                const wrapper = shallow(renderComponent({animations: false, node: {...data, toggled: false}}));
-                const instance = wrapper.instance();
-                const animations = instance.animations();
-                expect(animations).toEqual({
-                    toggle: {
-                        animation: {rotateZ: 0},
-                        duration: 0
-                    }
+
+    describe('animations', () => {
```

Comment:
@maximilianoforlenza I think we could remove this `describe('animations', () => {`